cmd/jujud/agent/machine/manifolds.go

```diff
@@ -928,7 +936,24 @@ func IAASManifolds(config ManifoldsConfig) dependency.Manifolds {
 			Clock:                        config.Clock,
 			NewCredentialValidatorFacade: common.NewCredentialInvalidatorFacade,
 		}))),
-	})
+	}
+
+	if utilsfeatureflag.Enabled(feature.InstanceMutater) {
```

Comment:
Does this worker get started on a controller?

Reply:
Why are we putting it behind a feature flag here? I thought this work replaced what we had in 2.5?

Reply:
@howbazaar yeah, but we'll rip out the feature flag once all done, so we don't ever end up like previously - I thought that was the plan we took away from the sprint?

Reply:
Fair call.